### DISCUSSION TITLE
My characters keep texting 18+ stuffs or innaproite messages in 13+ discord server

### DISCUSSION THREAD

**Discussion by TerminaRAT** [drizzle-mizzle/CharacterAI-Discord-Bot#150] [Q&A]
Hello, I'd like to ask if there is any way to stop characters doing 18+ stuff :D I have a discord server for American Dragon: Jake Long fans. And people I got there are 13+ some also 18+ and I made Jake Ai and Rose Ai and I made a chatroom for these bots to text with each other. Sadly it started that these bots started writing 18+ stuff and I had to instantly lock it and only put 18+ feature roles for people able to access it. Which is a big shame. Thank you if you will be able to help! ^^
![image](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/147729447/2dfde35d-e2cb-427e-ab80-17baaa584cde)

**Comment by drizzle-mizzle** [comment]
Well, the best you can do is rewrite their personality that way so they would refuse to discuss things like this

**Comment by TerminaRAT** [comment]
Ah thanks. Thats nice idea :D even it will go off the lore but at least 
helps
---------- Původní e-mail ----------
Od: drizzle-mizzle ***@***.***>
Komu: drizzle-mizzle/CharacterAI-Discord-Bot <CharacterAI-Discord-Bot@
noreply.github.com>
Kopie: TerminaRAT ***@***.***>, Author ***@***.***>
Datum: 17. 10. 2023 9:16:14
Předmět: Re: [drizzle-mizzle/CharacterAI-Discord-Bot] My characters keep 
texting 18+ stuffs or innaproite messages in 13+ discord server (Discussion 
#150)
"


Well, the best you can do is rewrite their personality that way so they 
would refuse to discuss things like this

—
Reply to this email directly, view it on GitHub
(https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/discussions/150#discussioncomment-7300636)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/BDHCYJ2XZZT4OWZ4CSRW4L3X7YWDLAVCNFSM6AAAAAA55VECIWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGMBQGYZTM)
.
You are receiving this because you authored the thread. Message ID: <drizzle
***@***.***
.com>
"
